src/components/fields/edit/LatLongEditor.jsx

```diff
@@ -56,7 +92,13 @@ export default function LatLongEditor({
           value={currentLongitudeString}
           onChange={e => {
             const inputValue = e.target.value;
-            onChange([currentLatitude, parseFloat(inputValue)]);
+            const floatValue = parseFloat(inputValue);
+            if (Number.isNaN(floatValue)) {
+              onChange([currentLatitude, parseFloat(inputValue)]);
+            } else {
+              onChange([currentLatitude, null]);
```

Comment:
Maybe not.... Seems unlikely that you would make this mistake twice. Possibly disregard the above.